docs/approach.rst

```diff
@@ -215,12 +270,22 @@ models (referred to as :math:`\kappa` and :math:`\rho`, respectively).
 
 A decision tree is applied to :math:`\kappa`, :math:`\rho`, and other metrics in order to
 classify ICA components as TE-dependent (BOLD signal), TE-independent
-(non-BOLD noise), or neither (to be ignored).
+(non-BOLD noise), or neither (to be ignored). 
+These classifications are saved in 
+`comp_table_ica.txt`.
 The actual decision tree is dependent on the component selection algorithm employed.
 ``tedana`` includes two options: `kundu_v2_5` (which uses hardcoded thresholds
 applied to each of the metrics) and `kundu_v3_2` (which trains a classifier to
```

Comment:
Can we remove this mention of kundu_v3 ? somehow it was leftover from previous docs ! We no longer offer this.

Reply:
banished